modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,104 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function() {
+
+     /* If no selected => select first */
+     if ($('.master-tabs a[aria-selected]').length === 0) {
+         $('.master-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.primary-tabs a[aria-selected]').length === 0) {
+         $('.primary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.secondary-tabs a[aria-selected]').length === 0) {
+         $('.secondary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.content-tabs a[aria-selected]').length === 0) {
+         $('.content-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     }
+
+     /* Click on a link */
+     $('body').on('click', '.master-tabs,.primary-tabs,.secondary-tabs,.content-tabs', function(event) {
+
+         let $target = $(event.target);
+         let tabNameClick = '';
+
+         if ($target.hasClass('master-tabs')) {
+             tabNameClick = '.master-tabs';
+         } else if ($target.hasClass('primary-tabs')) {
+             tabNameClick = '.primary-tabs';
+         } else if ($target.hasClass('secondary-tabs')) {
+             tabNameClick = '.secondary-tabs';
+         } else if ($target.hasClass('content-tabs')) {
+             tabNameClick = '.content-tabs';
+         }
+
+         // remove aria selected on all links + remove focusable
+         $(tabNameClick + ' a').attr('aria-selected', 'false');
+
+         // add aria selected on $this + focusable
+         $(tabNameClick + ' a.focus-visible').attr('aria-selected', 'true');
+
+         // Important - Must be set to true for October to set class="active" in the <li>
+         return true;
+     });
+
+     /* Keyboard arrow selection */
+     $('body').on('keydown', '.primary-tabs,.secondary-tabs,.master-tabs,.content-tabs', function(event) {
+
+         let $target = $(event.target);
```

Comment:
@ayumihamsaki could you also change `$(event.target)` -> `$(event.currentTarget)`. 
Mainly your target of keydown or click event would be span or anchor, and you need parent element.

Screenshots:
<img width="437" alt="Screenshot 2019-07-14 at 23 58 31" src="https://user-images.githubusercontent.com/1053320/61189239-56e8a000-a693-11e9-9c26-f0250c541914.png">
<img width="433" alt="Screenshot 2019-07-14 at 23 58 41" src="https://user-images.githubusercontent.com/1053320/61189240-56e8a000-a693-11e9-854f-f567b487ea86.png">

Reply:
changed.